R/makeTreeSummarizedExperimentFromPhyloseq.R

```diff
@@ -57,7 +62,7 @@ makeTreeSummarizedExperimentFromPhyloseq <- function(obj) {
         referenceSeq <- NULL
     }
     TreeSummarizedExperiment(assays = assays,
-                             rowData = obj@tax_table@.Data,
+                             rowData = rowData,
```

Comment:
a bug

Reply:
Not a bug

Reply:
How does taxonomic information otherwise pass from the phylosed to TSE object? And the direct slot access wasn't my first choice, but is faster than using the accessors, because they do stuff slowly.

Reply:
Ok, I thought that rowData = NULL leads to an error

I will fix that

Reply:
hmm. you might be right for that aspect. Can you combine the data if present?

Reply:
I remembered wrongly, `colData = NULL` leads to an error, but `rowData = NULL` not

```
> library(mia)
> data("GlobalPatterns")
> tse <- GlobalPatterns
> TreeSummarizedExperiment(assays = assays(tse), rowData = NULL)
class: TreeSummarizedExperiment 
dim: 19216 26 
metadata(0):
assays(1): counts
rownames(19216): 549322 522457 ... 200359 271582
rowData names(0):
colnames(26): CL3 CC1 ... Even2 Even3
colData names(0):
reducedDimNames(0):
mainExpName: NULL
altExpNames(0):
rowLinks: NULL
rowTree: NULL
colLinks: NULL
colTree: NULL
> TreeSummarizedExperiment(assays = assays(tse), colData = NULL)
Error in `rownames<-`(`*tmp*`, value = .get_colnames_from_assays(assays)) : 
  invalid rownames length
```

We could keep it as it is in master, but in this PR data is combined if it is present. Otherwise, empty DataFrame is created. 
This slot access thing is used in that part, 

(Actually that data gathering if present was already there but it was overwritten in this `TreeSE` constructor part)

![image](https://user-images.githubusercontent.com/60338854/131617870-f1c7a494-1e65-433e-bcbd-e4e4975c578a.png)